js/data/bucket/symbol_bucket.js

```diff
@@ -358,7 +366,7 @@ class SymbolBucket {
             // the buffers for both tiles and clipped to tile boundaries at draw time.
             const addToBuffers = inside || mayOverlap;
             this.addSymbolInstance(anchor, line, shapedTextOrientations, shapedIcon, this.layers[0],
-                addToBuffers, this.symbolInstancesArray.length, this.collisionBoxArray, feature.index, feature.sourceLayerIndex, this.index,
```

Comment:
@lucaswoj I believe it never was used, but rather was there on the hypothesis that it might be needed during cross-source placement.  @mollymerp does that sound right to you?